### PR TITLE
Renderers can have config

### PR DESCRIPTION
**Pull request by msp**
This means we can have a configurable colSpan (translates to Bootstrap col-md-XX) so we can determine widget widths.

I also hooked up a bunch more KPI widgets.

### CODE REVIEW

File: src/Data/Widget/Adapters/MetricAdapter.elm [hunk 1]
```diff
@@ -8,6 +8,9 @@ import Dict exposing (Dict)
 import Json.Decode as Json exposing (Value)
 
 
+-- TODO These are flipped (y, x) and zero based which is unique amongst the adapters
+
```
Comment (robwebdev) [inline src/Data/Widget/Adapters/MetricAdapter.elm:12]: D'oh!

**Review by robwebdev — APPROVED** [review]
Looks good. I'll have some rebasing to do :)

File: src/Data/Widget/Adapters/MetricAdapter.elm (hunk 1 above)
Comment (msp) [reply src/Data/Widget/Adapters/MetricAdapter.elm:12]: To be fair, I think you did this first and I broke rank. But (x, y) makes more sense to me. The zero/one based we could argue forever ;) 

Did confuse me a bit until I unpicked it tho.

What I'd really like, having now spent some time writing config for these widgets, is to be able to express the range same as Sheets e.g. A1:L9 

Rainy day job.

Comment (robwebdev) [reply src/Data/Widget/Adapters/MetricAdapter.elm:12]: `A1:L9` 👍 🥇 

(x,y) Also makes most sense to me as well, I must have been having a moment.  

"Along the corridor and up the stairs" and all that.

**Pull request merged**